apsuite/commissioning_scripts/meas_coupling_tune.py

```diff
@@ -0,0 +1,169 @@
+"""Coupling Measurement from Minimal Tune Separation."""
+
+import numpy as _np
+import matplotlib.pyplot as _plt
+
+
+from ..optimization import SimulAnneal as _SimulAnneal
+
+
+class FitTunes(_SimulAnneal):
+    """Tune Fit.
+
+    tunex = coeff1 * quad_parameter + offset1
+    tuney = coeff2 * quad_parameter + offset1
```

Comment:
Isn't it:  tuney = coeff2 * quad_parameter + **offset2** ?

Reply:
It is! thanks!